kloppy/domain/models/common.py

```diff
@@ -294,6 +304,240 @@ def duration(self):
+        raise NotImplementedError
+
+
+@dataclass
+class KloppyCoordinateSystem(CoordinateSystem):
+    @property
+    def provider(self) -> Provider:
+        return Provider.KLOPPY
+
+    @property
+    def origin(self) -> Origin:
+        return Origin.TOP_LEFT
+
+    @property
+    def vertical_orientation(self) -> VerticalOrientation:
+        return VerticalOrientation.TOP_TO_BOTTOM
+
+    @property
+    def pitch_dimensions(self) -> PitchDimensions:
+        return PitchDimensions(
+            x_dim=Dimension(0, 1),
+            y_dim=Dimension(0, 1),
+            x_per_meter=1 / self.length,
+            y_per_meter=1 / self.width,
+        )
+
+
+@dataclass
+class MetricaCoordinateSystem(CoordinateSystem):
+    @property
+    def provider(self) -> Provider:
+        return Provider.METRICA
+
+    @property
+    def origin(self) -> Origin:
+        return Origin.TOP_LEFT
+
+    @property
+    def vertical_orientation(self) -> VerticalOrientation:
+        return VerticalOrientation.TOP_TO_BOTTOM
+
+    @property
+    def pitch_dimensions(self) -> PitchDimensions:
+        return PitchDimensions(
+            x_dim=Dimension(0, 1),
+            y_dim=Dimension(0, 1),
+            x_per_meter=1 / self.length,
+            y_per_meter=1 / self.width,
+        )
+
+
+@dataclass
+class TracabCoordinateSystem(CoordinateSystem):
+    @property
+    def provider(self) -> Provider:
+        return Provider.TRACAB
+
+    @property
+    def origin(self) -> Origin:
+        return Origin.CENTER
+
+    @property
+    def vertical_orientation(self) -> VerticalOrientation:
+        return VerticalOrientation.BOTTOM_TO_TOP
+
+    @property
+    def pitch_dimensions(self) -> PitchDimensions:
+        return PitchDimensions(
+            x_dim=Dimension(-1 * self.length * 100 / 2, self.length * 100 / 2),
+            y_dim=Dimension(-1 * self.width * 100 / 2, self.width * 100 / 2),
+            x_per_meter=10000,
+            y_per_meter=10000,
+        )
+
+
+@dataclass
+class OptaCoordinateSystem(CoordinateSystem):
+    @property
+    def provider(self) -> Provider:
+        return Provider.OPTA
+
+    @property
+    def origin(self) -> Origin:
+        return Origin.BOTTOM_LEFT
+
+    @property
+    def vertical_orientation(self) -> VerticalOrientation:
+        return VerticalOrientation.BOTTOM_TO_TOP
+
+    @property
+    def pitch_dimensions(self) -> PitchDimensions:
+        return PitchDimensions(
+            x_dim=Dimension(0, 100),
+            y_dim=Dimension(0, 100),
+            x_per_meter=100 / self.length,
+            y_per_meter=100 / self.width,
+        )
+
+
+@dataclass
+class SportecCoordinateSystem(CoordinateSystem):
+    @property
+    def provider(self) -> Provider:
+        return Provider.SPORTEC
+
+    @property
+    def origin(self) -> Origin:
+        return Origin.BOTTOM_LEFT
+
+    @property
+    def vertical_orientation(self) -> VerticalOrientation:
+        return VerticalOrientation.TOP_TO_BOTTOM
```

Comment:
@koenvo I don't know what is the correct coordinate system for Sportec. Can you provide that info?

Reply:
@koenvo just a reminder that I need this info!